README.md

```diff
@@ -140,6 +140,7 @@ The `make run-iqe` command by default will run the smoke tests. However, if you
         --insights-upload UPLOAD_URL            optional, Use local directory path to populate a
                                                 "local upload directory".
         --ros-ocp-info                          Optional, Generate ROS for Openshift data.
+        --constant-values-ros-ocp               Optional, Generate constant values for ROS for OpenShift data.
```

Comment:
may be add info that this param won't work without ros-ocp-info   param? E.g. `This param works only in combination with ros-ocp-info param`